provisioner/influx-series.js

```diff
@@ -36,6 +36,15 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
     // Both the bid and price will be the pre-safety factor number
     bid: base.stats.types.Number,
     price: base.stats.types.Number,
+    securityToken: base.stats.types.String,
+  },
+});
+
+// Store when a security token was retreived
+module.exports.securityTokenRetreived = new base.stats.Series({
+  name: 'AwsProvisioner.SecurityTokenRetreived',
+  columns: {
+    securityToken: base.stats.types.String,
```

Comment:
Why?
I don't like this... There is no reason to leak security tokens into influxdb... which has horrible security...
Both here and in `spotRequestsSubmitted`.

---

If you want to record time from spot request submitted to secret retrieved, that's great but we can do that in other ways, but looking at the `created` timestamp on `secret` entities. I realize they don't have one, but adding one shouldn't take many seconds.